libs/sdk-ui-dashboard/.i18nrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
These files should have the copyright header as well. Can you please check why they were not added/reported missing by ESLint?

Reply:
Its look like eslint ignore dot files by default.